internal/ui/etui/ephemeral_tui.go

```diff
@@ -73,16 +75,25 @@ func OutputToEphemeralTUI(workerErrs <-chan error, subscription *partybus.Subscr
 	ctx := context.Background()
 	grypeUIHandler := grypeUI.NewHandler()
 
-eventLoop:
+	var errResult error
 	for {
 		select {
-		case err := <-workerErrs:
+		case err, ok := <-workerErrs:
```

Comment:
GoLand underlined `workerErrs` with this warning: "Receive may block because of 'nil' channel". May be worth checking for `nil` ahead of time.

Reply:
[ditto](https://github.com/anchore/grype/pull/156#discussion_r492236148)